determined-ai/determined
# fix: show `-` for empty data in searches table [ET-749]

## Ticket

[ET-749]


## Description




## Test Plan


Make sure Flat Runs feature flag is on. Go into any project with a significant number of experiments (Uncategorized is a good default choice). Perform the following steps on both the Runs and Searches tabs.

- Turn on all columns using column picker
- Check that in every cell where no data is available, there is a `-` present

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[ET-749]: https://hpe-aiatscale.atlassian.net/browse/ET-749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
